README.md

```diff
@@ -6,62 +6,59 @@ Convert Highcharts.JS charts to static image files.
 
 Version 2.1.0 has a couple of breaking changes:
 
-- Log destinations must now exist before starting file logging
+- Log destinations must now exist before starting file logging.
 - The following options are now disabled by default:
   - `callback`
   - `resources`
   - `customCode`
 
-Disabled options can be enabled by adding the `--allowCodeExecution` flag when
-starting the server/CLI. Using this flag is not recommended, and should not be
-done unless the server is sandboxed and not reachable on the public internet, or if only using the CLI in a controlled manner (e.g. it's not possible for a user to change the configuration sent to it through a different system).
```

Comment:
I see that the lines were shorter in the previous version of the readme. I'm not saying that it's a problem, but I've just noticed that there is no "max line length" in the current version (and for this reason some of the lines are very long).